synapse/tests/test_lib_modelrev.py

```diff
@@ -125,3 +127,55 @@ async def test_modelrev_0_2_6(self):
             self.eq(nodes[0].ndef[1], 'cve-2013-9999')
             self.eq(nodes[0].get('desc'), 'some words')
             self.eq(nodes[0].get('references'), (url3, url1, url0, url2))
+
+    async def test_modelrev_0_2_7_mirror(self):
+
+        regr = os.getenv('SYN_REGRESSION_REPO')
```

Comment:
Couldn't these just be calls to `getRegrDir` ?

Reply:
getRegrDir uses a temp dir and I wanted to be able to restart the cortex

Reply:
updated